docs/user_guide/installation.md

```diff
@@ -0,0 +1,293 @@
+
+## Installation Options
+
+There are several ways to deploy the geOrchestra Gateway:
+
+1. Docker container (recommended)
+2. Debian package
+3. JAR file (Spring Boot application)
+
+## Docker Installation
+
+!!! warning "Development Docker Compose Files"
+    Note that the `docker-compose.yml` and `docker-compose-preauth.yaml` files in the repository root are for **development purposes only** and should not be used for production deployments. For production, use the official geOrchestra Docker Compose project as described below.
+
+### Using the geOrchestra Docker Compose Project
+
+The recommended way to deploy the Gateway is using the official [geOrchestra Docker Compose project](https://github.com/georchestra/docker):
+
+```bash
+# Clone the repository
+git clone https://github.com/georchestra/docker.git
+cd docker
+
+# Start the full stack (including the gateway)
+docker-compose up -d
+```
+
+This will deploy the Gateway along with all required services (LDAP, PostgreSQL, etc.) and other geOrchestra components.
+
+### Custom Docker Compose Setup
+
+For a more targeted setup focusing just on the Gateway:
+
+```yaml
+# docker-compose.yml
+version: '3'
+services:
+  ldap:
+    image: georchestra/ldap:latest
+    environment:
+      - SLAPD_ORGANISATION=geOrchestra
+      - SLAPD_DOMAIN=georchestra.org
+      - SLAPD_PASSWORD=secret
+    volumes:
+      - ldap_data:/var/lib/ldap
+      - ldap_config:/etc/ldap
+
+  database:
+    image: postgres:14
+    environment:
+      - POSTGRES_USER=georchestra
+      - POSTGRES_PASSWORD=georchestra
+      - POSTGRES_DB=georchestra
+    volumes:
+      - pg_data:/var/lib/postgresql/data
+
+  gateway:
+    image: georchestra/gateway:latest
+    depends_on:
+      - database
+      - ldap
+    volumes:
+      - georchestra_datadir:/etc/georchestra
+    environment:
+      - JAVA_TOOL_OPTIONS=-Dgeorchestra.datadir=/etc/georchestra
+    env_file:
+      - .envs-common
+      - .envs-ldap
+      - .envs-database
+    ports:
+      - "8080:8080"
+    restart: unless-stopped
+
+volumes:
+  pg_data:
+  ldap_data:
+  ldap_config:
+  georchestra_datadir:
+```
+
+The `.envs-common` file should contain:
+
+```
+FQDN=georchestra-127-0-0-1.nip.io
+```
+
+The `.envs-ldap` file should contain LDAP connection information:
+
+```
+LDAPHOST=ldap
+LDAPPORT=389
+LDAPSCHEME=ldap
+LDAPADMINPASSWORD=secret
+```
+
+Launch with:
+
+```bash
+docker-compose up -d
+```
+
+### Debugging the Gateway Container
+
+For debugging the Gateway, you can enable remote debugging by adding environment variables:
+
+```yaml
+gateway:
+  environment:
+    - JAVA_OPTIONS=-Dorg.eclipse.jetty.annotations.AnnotationParser.LEVEL=OFF -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=0.0.0.0:5005
+    - XMS=256M
+    - XMX=1G
+  ports:
+    - "5005:5005"  # Expose debug port
+```
+
+### Using Kubernetes
```

Comment:
Kubernetes in the docker section ? I'd rather have it in a separate section. And would mention the geOrchestra Helm chart

Reply:
yeah, doesn't matter that's it's docker, they're two completely separate deployment models. Check out [these changes](https://github.com/georchestra/georchestra-gateway/compare/d1b355eda77aaf88691f5fa1a2f49c8a3a6323b0..d096bea6af1d0cb0c2279e213e0f3bcb0463ab87) or https://groldan.github.io/georchestra-gateway/user_guide/installation/#installation-options